### DISCUSSION TITLE
[BUG] can't locate some packages

### DISCUSSION THREAD

**Discussion by kckoo84** [cagnulein/qdomyos-zwift#332] [Q&A]
**Describe the bug**
On raspberry pi3, After  sudo apt update && sudo apt upgrade,
and after sudo apt install git libqt5bluetooth5 libqt5widgets5 libqt5positioning5 libqt5xml5 qtconnectivity5-dev qtpositioning5-dev libqt5charts5-dev libqt5charts5 qt5-default libqt5networkauth5-dev,
it gives me this

E: Unable to locate package libqt5charts5-dev
E: Unable to locate package libqt5charts5
E: Unable to locate package libqt5networkauth5-dev

seems like qt5 version from the default is 5.7.1 and looks does not have those 3 packages.
Would you be able to suggest what need to be done to get those 3 packages ?

**Comment by cagnulein** [comment]
@kckoo84 which distro are you using? 5.7.1 seems very old to me. You should have 5.11 at least on raspbian
